udata/migrations/2020-06-11-add-resource-fs-filename.py

```diff
@@ -0,0 +1,36 @@
+'''
+The purpose here is to fill every resource with a fs_filename string field.
+The migration iterates on every file in the ressource storage.
+It extracts the dataset's slug from the file path, retrieves the dataset,
+and iterates on the dataset's resources to find the one owning the file.
+If one is found the fs_filename field is filled.
+If none is found the file is deleted.
+'''
+import logging
+
+from udata.core import storages
+from udata.models import Dataset, Resource
+
+log = logging.getLogger(__name__)
+
+
+def migrate(db):
```

Comment:
Maybe add some counters at the end: x resources files deleted, y logos, z resources updated...

Reply:
👍